calico_containers/pycalico/datastore.py

```diff
@@ -326,11 +326,13 @@ def get_host_bgp_ips(self, hostname):
             return (ipv4, ipv6)
 
     @handle_errors
-    def get_ip_pools(self, version):
+    def get_ip_pools(self, version, ipam_only=False):
```

Comment:
Why have ipam only in one place and skip ipam in another? Would it be better to be consistent?